app/models/project_metadata.rb

```diff
@@ -58,17 +74,47 @@ def create( params:)
     project.metadata["project_id"]
   end
 
+  def required_keys
+    tableized = required_field_labels.map { |v| v.parameterize.underscore }
+    tableized.map(&:to_sym)
```

Comment:
You probably could avoid the second call to `.map` if you update the first one to do `v.parameterize.underscore.to_sym`. No big deal either way.